tests/recorded/replace_builtin_empty_list.txt

```diff
@@ -0,0 +1,11 @@
+list({})
+list([])
+list("")
+list(())
```

Comment:
Let's also add cases (in this file, so it's obvious on inspection) to demonstrate that this *doesn't* happen with non-empty collections.